disvae/models/losses.py

```diff
@@ -235,8 +238,24 @@ def __call__(self, data, model, optimizer, storer):
             vae_loss = rec_loss + kl_loss + self.gamma * tc_loss
         else:
             # return vae loss without mutual information term
+            # change latent dist to torch.tensor (could probably avoid this)
+            latent_dist = torch.stack((latent_dist[0], latent_dist[1]), dim=2)
+            # calculate log p(z)
+            prior_params = torch.zeros(half_batch_size, latent_dist.size(1), 2)
```

Comment:
that will break on GPU no ? .to(device)

Reply:
I have fixed this, now running on GPU as well.